apps/web/src/pages/Applications/ApplicationReviewPage/ApplicationReviewPage.tsx

```diff
@@ -616,7 +616,9 @@ const ApplicationReview = ({
 export const Component = () => {
   const { application } = useApplication();
 
-  const experiences: Experience[] = unpackMaybes(application.user.experiences);
+  const experiences: Omit<Experience, "user">[] = unpackMaybes(
```

Comment:
Not sure, it might be better to drop the user field from the `Experience` type instead? Given you query in the direction of user -> experience and never the other way around. 
And then after that get rid of all the `Omit`

Reply:
I think you may be right. Is there ever a scenario where we need to get the user from an experience? Seems like what you mentioned is true and we only really ever query experiences from a user.

@tristan-orourke How do you feel about this? I wouldn't say we need to do that here but maybe we can create a ticket to remove `experience.user` from the schema? :thinking:

Reply:
It may also be the case for a few other types. 
I think we go user -> userSkills but never userSkill -> user

Reply:
Yeah, I tihnk a lot of the entities that are part of a users profile do not need to include the user. They would be useful in the event that we wanted to see all users who claim a specific skill but right now we aren't doing that and I can't ever think of a time where we need to get the user from a specific experience :thinking:

Reply:
Yeah, we've talked about wanted to remove this kinds of relation fields from the schema. Especially if we're not already using them!
Let's do it. Could you make a ticket?

Reply:
Thanks, created.

https://github.com/GCTC-NTGC/gc-digital-talent/issues/11230